types/primitive.yaml

```diff
@@ -52,3 +52,8 @@ Signature:
   type: string
   pattern: "^0x[a-fA-F0-9]{192}$"
   example: "0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505"
+
+Uint8:
+  type: string
+  pattern: "^0x[a-fA-F0-9]{2}$"
+  example: "0xff"
```

Comment:
Hex? Aren't we using stringified decimal numbers?

Reply:
I wasn't sure here, they're flags which would make more sense in hex, I could change back to stringified numbers if thats preferred? not sure if its easy to specify limits that way, but I can try again?